OlhaSvyrydenko/HW3/03.1 Practical tasks.py

```diff
@@ -0,0 +1,65 @@
+#1st TASK
+
+string = ''' The Zen of Python, by Tim Peters:
+
+    Beautiful is better than ugly.
+    Explicit is better than implicit.
+    Simple is better than complex.
+    Complex is better than complicated.
+    Flat is better than nested.
+    Sparse is better than dense.
+    Readability counts.
+    Special cases aren't special enough to break the rules.
+    Although practicality beats purity.
+    Errors should never pass silently.
+    Unless explicitly silenced.
+    In the face of ambiguity, refuse the temptation to guess.
+    There should be one-- and preferably only one --obvious way to do it.
+    Although that way may not be obvious at first unless you're Dutch.
+    Now is better than never.
+    Although never is often better than right now.
+    If the implementation is hard to explain, it's a bad idea.
+    If the implementation is easy to explain, it may be a good idea.
+    Namespaces are one honking great idea -- let's do more of those! 
+    '''
+
+#find separately the number of occurence of the words "better", "never", "is". 
+#print( "Number of occurence of better:", string.count("better"))
+#print("Number of occurence of never:", string.count("never")) 
+#print("Number of occurence of is:", string.count('is'))
+#print("Number of occrrence of better:", string.count('better'), ", of never:", string.count('never'), ", of is:", string.count('is'))
+
+# display all text in appercase.
+#print(string.upper())
+
+#replace all occurence of the symbol "i" with "&"
+#replaced_string = string.replace('i', '&')
+#print(replaced_string)
+
+#2nd TASK
```

Comment:
excellent idea
but look at more simpler implementation:
digit_number = input("Enter a four digit number:")

product =  int(digit_number[0]) * int(digit_number[1]) * int(digit_number[2]) * int(digit_number[3])
print(f"{digit_number[0]} * {digit_number[1]} * {digit_number[2]} * {digit_number[3]} = ", product)
print()

revers = int(''.join(reversed(digit_number)))
print(f"{digit_number} in revers ordef = ", revers)
print()

sort = int("".join(sorted(digit_number)))
print(f"{digit_number} sort =", sort)